nats-base-client/jsutil.ts

```diff
@@ -89,7 +89,7 @@ export function checkJsError(msg: Msg): NatsError | null {
   if (!h) {
     return null;
   }
-  return checkJsErrorCode(h.code, h.status);
+  return checkJsErrorCode(h.code, h.description);
```

Comment:
We want to check description vs status?

Reply:
the status line contains the code/message - the description is just the message part of that combo, so it uglified some of the errors.